scripts/deploy_local.sh

```diff
@@ -21,7 +21,7 @@ cd $PANTHEON_CODEBASE
 set -e
 # build the distribution
 # build the karaf-features and karaf-config along with pantheon modules
-./mvnw clean install -U -pl sling-org-apache-sling-karaf-features,sling-org-apache-sling-karaf-configs,pantheon-bundle,pantheon-karaf-feature,pantheon-karaf-dist
+./mvnw clean install -U -e  -pl sling-org-apache-sling-karaf-features,sling-org-apache-sling-karaf-configs,pantheon-bundle,pantheon-karaf-feature,pantheon-karaf-dist
```

Comment:
Do we really need to change current build command ?

Reply:
Reverted the change.